src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  margin: 0;
+  padding: 0;
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 300;
+}
+
+.search {
+  display: block;
+  position: relative;
+  height: 70px;
+  margin-top: 20px;
```

Comment:
You should avoid margin collapsing, when a child margins go through a parent and push off another parent
[useful_link](https://www.smashingmagazine.com/2019/07/margins-in-css/#parent-and-first-or-last-child-element)
So, if you want one element to be push off from another element use margins at THESE elements, not their children